src/ProjectNeon/Assets/Data/Enemies/AI/DumbAI.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DumbAI : TurnAI
+{
+    /**
+     * The AI member
+     */
+    private Member me;
+
+    /**
+     * This AI allied members.
+     */
+    private List<Member> allies;
```

Comment:
We should bind to `BattleState` in order to query enemies/allies.

If needed, you can update that to return members.